modules/tinymce/src/core/main/ts/api/file/BlobCache.ts

```diff
@@ -75,7 +76,7 @@ export const BlobCache = (): BlobCache => {
   };
 
   const toBlobInfo = (o: BlobInfoData): BlobInfo => {
-    if (!o.blob || !o.base64) {
+    if (!o.blob || (!o.base64 && !o.allowEmptyFile)) {
```

Comment:
Please make this a proper empty string check. The code was written before we had types, when it was possible to have an undefined property; now we don't accept truthy checks for empty string validation.

Reply:
ok, I think I understood, have a check